pipelines/tekton/azureml-container-pipeline/model-upload/basic-pod.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: basic-pod
```

Comment:
I feel like this name is a bit too generic, maybe `local-model-to-pv-pod` or something like that would be better.

Reply:
👍